src/Options/Definitions.js

```diff
@@ -175,6 +175,12 @@ module.exports.ParseServerOptions = {
     action: parsers.booleanParser,
     default: true,
   },
+  disableCollation: {
```

Comment:
To fix the failing definitions test, see https://github.com/parse-community/parse-server/blob/alpha/CONTRIBUTING.md#parse-server-configuration